DeAceroDLabs/marketplace-frontend
# [DMP-657] Fix total results in search

## What is done

[[DMP-657] Página de Search no funciona](https://deacero.atlassian.net/browse/{DMP-657})

## How is done

1. Most of the changes were done in the backend
2. Added totalResults to the frontend to display as total results instead of using array length
3. Modified statement so page won't crash in case of image array missing in data

## How to test

1. Run the project via `npm start`
4. Head over to search with 'Clavos' and there should be 40 total results
5. Search with 'Clavos A' and there should be 63 total results

## Evidence

=========================================BEFORE FIX=================================
[screen-capture (2).webm](https://user-images.githubusercontent.com/116199447/199820718-4b41b2e4-6680-42c3-a4bb-3103a43bd043.webm)




=========================================AFTER FIX==================================
![screen-capture-_3_](https://user-images.githubusercontent.com/116199447/199819613-1c6d108a-de1c-450e-a5c7-47d391732356.gif)
